tutorials/create-hello-world-mta/create-hello-world-mta.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Create a Hello World Multitarget Application
+description: This tutorial will help you create a module that describes a Java application, a resource that describes your database binding and additional metadata required for the deployment.
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, programming-tool>java]
+primary_tag: software-product>sap-business-technology-platform
+---
+
+## Prerequisites
+ - You have created a Java application `.war file.`
+ - You have an SAP BTP database configured in your subaccount and valid credentials.
+
+## Details
+### You will learn
+  - How to create a module that describes a Java application
+  - How to create a resource that describes your database binding
+  - How to create additional metadata required for the deployment
+
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Modeling the MTA deployment descriptor)]
+
+You define the entities that have to be deployed, namely modules and resources related to other modules, in the MTA deployment descriptor. This is a YAML file that defines the contract between you as a deployable artifact provider and the SAP BTP as a deployer tool. Initially, you have to describe the metadata required for the deployment.
+
+1. Create an empty text file and name it `mtad.yaml.`
+2. Using a text editor, enter the following data in the file:
+    >
+    > #### Note
```

Comment:
You do not need to write out the word note.  When using the ">" just write what you want to say.  This will offset visually and the word "note" is redundant.